docs/core/tools/global-tools-how-to-use.md

```diff
@@ -0,0 +1,115 @@
+---
+title: How to install and use a .NET Core global tool
+description: Learn how to install and use a .NET tool as a global tool.
+author: tdykstra
+ms.author: tdykstra
+ms.date: 02/12/2020
+---
+
+# Install and use a .NET Core global tool using the .NET Core CLI
+
+**This article applies to:** ✔️ .NET Core 3.x SDK
+
+This tutorial teaches you how to install and use a global tool. You use a tool that you create in the [first tutorial of this series](global-tools-how-to-create.md).
+
+## Prerequisites
+
+* Complete the [first tutorial of this series](global-tools-how-to-create.md).
+
+## Use the tool as a global tool
+
+1. Install the tool from the package by running the [dotnet tool install](dotnet-tool-install.md) command in the *botsay* project folder:
+
+   ```dotnetcli
+   dotnet tool install --global --add-source ./nupkg botsay
+   ```
+
+   The `--global` parameter tells the .NET Core CLI to install the tool binaries in a default location that is automatically added to the PATH environment variable.
+
+   The `--add-source` parameter tells the .NET Core CLI to temporarily use the `./nupkg` directory as an additional source feed for NuGet packages.
+
+   The output shows the command used to call the tool and the version installed:
+
+   ```console
+   You can invoke the tool using the following command: botsay
+   Tool 'botsay' (version '1.0.0') was successfully installed.
+   ```
+
+   > [!NOTE]
```

Comment:
I would move this note block into step 2, below the code block, and say "If this step fails...".